lib/rails/dom/testing/railtie.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Rails
+  module Dom
+    module Testing
+      class Railtie < Rails::Railtie # :nodoc:
+        initializer "rails_dom_testing.default_html_version" do |app|
+          if app.config.respond_to?(:dom_testing_default_html_version) # Rails 7.1+
+            version = app.config.dom_testing_default_html_version
+            Rails::Dom::Testing.default_html_version = version if version
+          end
```

Comment:
How about:
```suggestion
          version = app.config.try(:dom_testing_default_html_version) # Rails 7.1+
          Rails::Dom::Testing.default_html_version = version if version
```

Not sure if the usage of `Object#try` has fallen out of fashion, but it seems like the exact use-case for it.

Reply:
Done.